### ISSUE TITLE
Why Loss function is 'MSE'?

### ISSUE THREAD

**Issue by ahmed-nady** [faustomorales/keras-ocr#93]
Firstly, thanks for your re-implementing CRAFT text detector. I know that I should prepare my dataset so that I provide the detector with the char box annotation to fine-tune. but I ask for choosing the loss function. I saw you choose the mean squared error (MSE), why? and can I choose another loss function?

**Comment by faustomorales** [comment]
Hi there -- Of course, please do choose a different loss function, as desired. You can do this using `detector.model.compile()`. And feel free to file a PR to change the default if you find a different loss function provides you with better results. Thanks!
